src/config/mongoose_config_spec.erl

```diff
@@ -554,27 +556,47 @@ wpool_items() ->
                                     validate = positive}
      }.
 
+wpool_defaults(<<"cassandra">>) ->
+    maps:merge(wpool_defaults(), #{<<"workers">> => 20});
```

Comment:
The docs are wrong about it, the default is `100` there.

Reply:
Good catch!